conans/client/graph/proxy.py

```diff
@@ -113,6 +113,11 @@ def _download_recipe(self, layout, ref, output, remotes, remote, recorder):
         def _retrieve_from_remote(the_remote):
             output.info("Trying with '%s'..." % the_remote.name)
             # If incomplete, resolve the latest in server
+            if "https://conan.bintray.com" in the_remote.url:
```

Comment:
When Bintray, if you clicked the `set me up` button it didn't show you the `https://conan.bintray.com/...` URL, but `https://bintray.com/conan....` (I cannot remember exactly the path)... and many people had that one configured. Do you remember? Do we want to check also for that URL?

Reply:
Yes, I'd say to check fur ``bintray.com``, that should be good enough for both cases, isn't it?

Reply:
the URL `https://bintray.com/conan....` will be useless as it is not working (only `conan.bintray.com` is still operational), so checking for it doesn't make sense as it won't work from the beginning and would never get this warning message

Reply:
This is a very good reason indeed 😄